docs/reference/troubleshooting/common-issues/task-queue-backlog.asciidoc

```diff
@@ -1,51 +1,90 @@
-and long running tasks can all contribute to a backlogged task queue.
+A backlogged task queue can prevent tasks from completing and put the cluster
+into an unhealthy state. Resource constraints, a large number of tasks being
+triggered at once, and long running tasks can all contribute to a backlogged
+task queue.
 
 [discrete]
 [[diagnose-task-queue-backlog]]
 ==== Diagnose a task queue backlog
 
 **Check the thread pool status**
 
-A <<high-cpu-usage,depleted thread pool>> can result in <<rejected-requests,rejected requests>>. 
+A <<high-cpu-usage,depleted thread pool>> can result in
+<<rejected-requests,rejected requests>>. This may surface restricted to a
+<<data-tiers,data tier>>'s traffic, potentially with <<hotspotting,hot spotting>>
+symptoms.
 
-You can use the <<cat-thread-pool,cat thread pool API>> to 
-see the number of active threads in each thread pool and
-how many tasks are queued, how many have been rejected, and how many have completed. 
+You can use the <<cat-thread-pool,cat thread pool API>> to see the number of
+active threads in each thread pool and how many tasks are queued, how many
+have been rejected, and how many have completed.
 
 [source,console]
 ----
 GET /_cat/thread_pool?v&s=t,n&h=type,name,node_name,active,queue,rejected,completed
 ----
 
+The `active` and `queue` statistics are instantaneous while the `rejected` and
+`completed` statistics are cumulative from node start up.
+
 **Inspect the hot threads on each node**
 
-If a particular thread pool queue is backed up, 
-you can periodically poll the <<cluster-nodes-hot-threads,Nodes hot threads>> API 
-to determine if the thread has sufficient 
-resources to progress and gauge how quickly it is progressing.
+If a particular thread pool queue is backed up, you can periodically poll the
+<<cluster-nodes-hot-threads,Nodes hot threads>> API to determine if the thread
+has sufficient resources to progress and gauge how quickly it is progressing.
 
 [source,console]
 ----
 GET /_nodes/hot_threads
 ----
 
-**Look for long running tasks**
+**Look for long running node tasks**
+
+Long-running tasks can also cause a backlog. You can use the <<tasks,task
+management>> API to get information about the node tasks that are running.
+Check the `running_time_in_nanos` to identify tasks that are taking an
+excessive amount of time to complete.
+
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true
+----
+
+If a particular `action` is suspected, you can filter in further. Most common are: 
+
+* <<docs-bulk,bulk index>> related
++
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true&actions=indices:data/write/bulk
+----
+
+* search related
++
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true&actions=indices:data/write/search
+----
+
+Note the API response may contain tasks columns `description` and `header` 
+which enable futher diagnosising task parameters, target, and requestor. 
```

Comment:
always avoid the phrase "note that" as it doesn't add anything. if something is especially noteworthy, you can use a note block to call it out in a little box.

`NOTE: blah blah`

```suggestion
The API response may contain additional tasks columns, including `description` and `header`, which provides the task parameters, target, and requestor. You can use this information to perform further diagnosis.
```